mocks/flask-react-mocks/server/sampleData.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+
+shortLoremIpsum = "Lorem id sint aliqua tempor tempor sit. Ad dolor dolor ut nulla mollit dolore non eiusmod Lorem tempor nisi cillum."
+longLoremIpsum = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Rhoncus dolor purus non enim praesent elementum facilisis leo vel. Risus at ultrices mi tempus imperdiet."
+
+sampleGridData = [
```

Comment:
This is the sample data for master detail actually,
Grid is actually:
```
sampleData.gridTextAssets = [
  {
    description: shortLoremIpsum,
    header: "Company A",
    id: 1
  },
  {
    description: shortLoremIpsum,
    header: "Company B",
    id: 2
  },
  {
    description: shortLoremIpsum,
    header: "Company C",
    id: 3
  },
  {
    description: shortLoremIpsum,
    header: "Company D",
    id: 4
  },
  {
    description: shortLoremIpsum,
    header: "Company E",
    id: 5
  },
  {
    description: shortLoremIpsum,
    header: "Company F",
    id: 6
  },
  {
    description: shortLoremIpsum,
    header: "Company G",
    id: 7
  },
  {
    description: shortLoremIpsum,
    header: "Company H",
    id: 8
  },
  {
    description: shortLoremIpsum,
    header: "Company I",
    id: 9
  }
];
```

Reply:
I looked at the grid component. It is actually using the property `gridTextAssets.shortDescription` instead of` `gridTextAssets.description`. Hence, the above data would not work for a grid component. Although you are right that it never uses the `longDescription` attribute which is only being used in the masterDetail page.

Reply:
you know what this PR made me realized we are no longer using gridTextAssets in the nodes backend. I will be making a PR to remove this from all merging. Thanks for noticing this!

Reply:
Or do you believe we should keep this sample data and instead change the frontend components?

Reply:
Ideally, I think we should be changing the front-end components because that would make more sense for any person.

Reply:
So you're saying we keep the sampleGridData that i provided (for node) and we change the frontend to use it rather than sharing the data between grid and master detail?

Reply:
Yeah, that is what I meant. However, I think for now, lets share the data between grid and master detail page. Changing this would mean changing a lot of things like the route in node.js and all front-end frameworks. I am not sure whether it would be an ideal thing to do now or should be incorporated once we have these frameworks (Vue and Flask in). It is just an opinion.

What do you think about it?

Reply:
Okay so I asked kai and he said clint wanted it to be shared between grid and master detail so im going to remove it from node in a seperate pr